frappe/core/doctype/doctype/doctype.py

```diff
@@ -222,7 +222,7 @@ def validate(self):
 
 		check_email_append_to(self)
 
-		if self.default_print_format and not self.custom:
+		if self.default_print_format and not self.custom and not frappe.session.user == "Administrator":
```

Comment:
This was probably done to prevent hardcoding a print format in code. Check the error message that we throw.